applications/RANSApplication/RANSApplication.py

```diff
@@ -6,11 +6,7 @@
 elif (IsDistributedRun()):
     raise Exception("Distributed run requires TrilinosApplication")
 
-if (CheckIfApplicationsAvailable("FluidDynamicsApplication")):
-    from KratosMultiphysics.FluidDynamicsApplication import *
-else:
-    raise Exception("RANSApplication requires FluidDynamicsApplication.")
-
+from KratosMultiphysics.FluidDynamicsApplication import *
```

Comment:
Can it be changed to `import KratosMultiphysics.FluidDynamicsApplication` instead of importing *?

Reply:
done :)